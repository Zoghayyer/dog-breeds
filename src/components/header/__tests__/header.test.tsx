import React from 'react';
import { shallow } from 'enzyme';
import Header, { HeaderProps } from '../header';

describe('(Component) Header', () => {
  let component;
  const defaultProps: HeaderProps = {
    children: <div>child</div>,
    className: 'className',
  };

  const shallowHeader = (extendedProps = {}) => {
    component = shallow(<Header {...defaultProps} {...extendedProps} />);
  };

  beforeEach(() => shallowHeader());

  it('renders without crashing', () => {
    shallow(<Header><div id="child" /></Header>);
    expect(component).toMatchSnapshot();
  });
});
