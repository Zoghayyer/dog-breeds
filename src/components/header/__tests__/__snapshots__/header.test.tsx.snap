// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Header renders without crashing 1`] = `
<div
  className="className"
>
  <div>
    child
  </div>
</div>
`;
